cobbler/tftpgen.py

```diff
@@ -864,10 +864,14 @@ def build_kernel_options(self, system, profile, distro, image, arch: str, autoin
                     autoinstall_path = "http://%s/cblr/svc/op/autoinstall/profile/%s" \
                                        % (httpserveraddress, profile.name)
 
-            if distro.breed is None or distro.breed == "redhat":
+            if distro.breed is None or distro.breed == "redhat" or distro.breed == "redhatlegacy":
 
-                append_line += " kssendmac"
-                append_line = "%s inst.ks=%s" % (append_line, autoinstall_path)
+                if distro.breed == "redhatlegacy":
+                    append_line += " kssendmac"
+                    append_line = "%s ks=%s" % (append_line, autoinstall_path)
+                else:
+                    append_line += " inst.ks.sendmac"
+                    append_line = "%s inst.ks=%s" % (append_line, autoinstall_path)
```

Comment:
Quite strange mix of `append_line += ...` and `append_line = "%s ... %s" % (append_line, ...)`
```suggestion
                if distro.breed == "redhatlegacy":
                    append_line += " kssendmac ks=%s" % autoinstall_path
                else:
                    append_line += " inst.ks.sendmac inst.ks=%s" % autoinstall_path
```

Reply:
The mix is what the code was doing before I touched it, so I left it this way and changed only kssendmac/inst.ks.sendmac, but that can indeed be made into one line.